src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx

```diff
@@ -15,22 +15,76 @@ redirects:
   - /docs/apm/apm-ui-pages/monitoring/apm-data-infrastructure
 ---
 
-import infrastructureInfraApmDataChart from 'images/infrastructure_screenshot-crop_infra-apm-data-chart.webp'
+import apmCriticalViolationsinApdex from 'images/apm_screenshot-full_critical-violations-in-Apdex.webp'
+
+import apmCPUPercentageViolationinAPM from 'images/apm_screenshot-crop_CPU-percentage-violation-in-APM.webp'
+
+import apmToggleBetweenHistogramViews from 'images/apm_screenshot-crop_toggle-between-histogram-views.webp'
+
+import apmExploringaSpecificHostinAPM from 'images/apm_screenshot-full_exploring-a-specific-host-in-APM.gif'
+
+import apmFilterbyApplication from 'images/apm_screenshot-full_filter-by-application.webp'
 
 The integration of [APM](/docs/apm/new-relic-apm/getting-started/welcome-new-relic-apm) and [infrastructure](/docs/infrastructure/new-relic-infrastructure/getting-started/welcome-new-relic-infrastructure) data lets you see your APM data and infrastructure data side by side so you can find the root cause of problems more quickly.
 
-The main ways to find and use APM data in infrastructure monitoring are:
+First, this doc will walk you through the process of resolving infrastructure issues with APM. Then it will dig deeper into some of the key features of APM and infrastructure monitoring. 
 
-* [View APM charts on infrastructure monitoring UI pages](#apm-chart)
-* [Filter hosts by application data](#filter-by-app)
-* [Switch between infrastructure and APM](#switch)
-* [Examine APM data in Inventory and Events pages](#inventory-events)
+<Steps>
+    <Step>
+    ## Do you have a problem in your system? 
```

Comment:
I'd err on the side of describing the `thing` to be solved for in the title, so users know what action theyre going to take. It looks like the action here is 'respond to an alert', so maybe just making that (or something like that) the title

Reply:
great! thanks :)